packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet.tsx

```diff
@@ -59,19 +72,6 @@ export class AtomicSmartSnippet implements InitializableComponent {
    * The [heading level](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) to use for the question at the top of the snippet, from 1 to 5.
    *
    * We recommend setting this property in order to improve accessibility.
-   *
-   * You can style the snippet by inserting a template element like this:
-   * ```html
```

Comment:
For whatever reason, I put that doc in this prop which has nothing to do with this example. I have no idea how I made that mistake...